src/aggregators/coevolve.jl

```diff
@@ -84,8 +84,13 @@ function (p::CoevolveJumpAggregation)(integrator::I) where {I <:
     nothing
 end
 
-function (p::CoevolveJumpAggregation{T, S, F1, F2})(integrator::AbstractSSAIntegrator) where
-    {T, S, F1, F2 <: Union{Tuple, Nothing}}
+function (p::CoevolveJumpAggregation{
+        T,
+        S,
+        F1,
+        F2,
+    })(integrator::AbstractSSAIntegrator) where
+        {T, S, F1, F2 <: Union{Tuple, Nothing}}
```

Comment:
@ChrisRackauckas @YingboMa this still seems like the expanding things unnecessarily to one per line bug is around... This is adding a lot of unnecessary lines with all the new line breaks for something that can fit on one line within the col width (i.e. the parametric types).

Reply:
Yeah this looks incorrect indeed.

Reply:
the bug here is `})` should be 4 spaces dedented.

the line expansion is due to the margin ruling and so it'll break the line based on the nesting style which is delegating to either Default and YAS styles in SciML depending on whether yas_style_nesting is true or false. Blue style tries the args on a line indented before splitting them up so perhaps replacing default style with blue style for nesting would be what you guys want. otherwise you can increase the margin.

Reply:
Defaulting to blue style's approach there sounds like a good approach.

Reply:
Doesn't blue style fully expand into 1-arg per line if all args don't fit in one line? I'd much prefer to not do that and try to minimize newly introduced lines by splitting.

Reply:
yes it does. 

https://github.com/domluna/JuliaFormatter.jl/blob/master/src/styles/blue/nest.jl#L1-L133

you could alter that to try a incremental line splitting thing. Try 1 line, if not 1, then 2, if not 2, then 3, etc.

in what situations do you want line splitting though? you can set the margin to be high if it's a global want, or alter some nesting functions to not split.

Reply:
fix for the bug here https://github.com/domluna/JuliaFormatter.jl/pull/791

Reply:
Using JuliaFormatter v1.0.45 I'm still getting a similar reformat to above:

<img width="1299" alt="Screenshot 2023-12-12 at 8 25 52 AM" src="https://github.com/SciML/JumpProcesses.jl/assets/9385167/80af06b5-ed60-46e9-b0ec-48810e7d4cdc">

Reply:
my email didn't send but the fix was for the extra indent